server/shared/src/main/scala/org/http4s/server/staticcontent/WebjarService.scala

```diff
@@ -30,8 +30,6 @@ import scala.util.control.NoStackTrace
 
 /** [[org.http4s.server.staticcontent.WebjarServiceBuilder]] builder
   *
-  * @param blocker execution context for blocking I/O
-  * @param filter To filter which assets from the webjars should be served
```

Comment:
This parameter is still here, just renamed :)

Reply:
It looks so. As an excuse - I was only about removing outdated params. But yeah, some scaladocs are needed solid refactoring.

Reply:
No excuses needed! You've been doing fantastic work cleaning all this up, let's not let it be thankless 😉